app/tests/e2e/test_connections.py

```diff
@@ -172,18 +172,24 @@ async def test_get_connections(
     finally:
         # clean up temp connection
         await alice_member_client.delete(f"{BASE_PATH}/{alice_connection_id}")
-        await bob_member_client.delete(f"{BASE_PATH}/{bob_connection_id}")
 
 
 @pytest.mark.anyio
 async def test_get_connection_by_id(
     bob_member_client: RichAsyncClient,
+    alice_member_client: RichAsyncClient,
 ):
-    invitation_response = await bob_member_client.post(f"{BASE_PATH}/create-invitation")
-    invitation = invitation_response.json()
-    connection_id = invitation["connection_id"]
+    connection_alias = "TempAliceBobConnectionById"
```

Comment:
We may as well add an assertion at the end, that the fetched record has this alias. Doesn't seem like alias is used otherwise.

Also, worth noting this test is run for regression tests too, in which case alice+bob members will keep gaining new records. We should therefore add a delete call for this connection record at the end of the test. Best way to do it is:

```py
# as it is currently, establish connection:
    bob_and_alice_connection = await create_bob_alice_connection(
        alice_member_client, bob_member_client, alias=connection_alias
    )

    bob_connection_id = bob_and_alice_connection.bob_connection_id

#Then start try block:
try:
... # get and assertions
finally:
... # call delete endpoint for this record
```

So that if anything goes wrong for whatever reason, the delete will always be called after connection created.